base_limiter.go

```diff
@@ -10,6 +11,11 @@ import (
 	"github.com/2manymws/rl/counter"
 )
 
+const (
+	remoteAddrKey = "remote_addr"
+	hostKey       = "host"
```

Comment:
How about exporting them and having users use them?

```suggestion
	RemoteAddrKey = "remote_addr"
	HostKey       = "host"
```

(Perhaps variable names when exporting might need to be considered.)